src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
```

Comment:
I think the format is sas_impl/$ADMINISTRATION/$SAS_IMPLEMENTATION and the $ADMINISTRATOR is the SAS-CA Certified unique SAS Administrator identifier